src/components/eRACommons.js

```diff
@@ -56,17 +54,16 @@ export const eRACommons = hh(class eRACommons extends React.Component {
     }
   };
 
-  getResearcherProperties = async () => {
-    const currentUserId = Storage.getCurrentUser().dacUserId;
-    Researcher.getPropertiesByResearcherId(currentUserId).then(response => {
-      const isAuthorized = JSON.parse(_.get(response, 'eraAuthorized', "false"));
-      const expirationCount = AuthenticateNIH.expirationCount(_.get(response, 'eraExpiration', 0));
+  getUserInfo = async () => {
+    User.getMe().then((response) => {
+      const isAuthorized = JSON.parse(_.get(response.researcherProperties, 'eraAuthorized', "false"));
+      const expirationCount = AuthenticateNIH.expirationCount(_.get(response.researcherProperties, 'eraExpiration', 0));
```

Comment:
Some points here:
* Here and elsewhere, we should move away from using `_` to refer to lodash functions and instead import them directly. 
* I also prefer the fp version as it is more legible and composable. 
* I don't think we need JSON.parse here, the object should already be a json structure. 
* We're not parsing the user's properties correctly here. There are no properties with the names as described, each property has the form of `{"propertyId":2579,"userId":3351,"propertyKey":"eraAuthorized","propertyValue":"true"}` so you have to look for the `propertyKey` key in each property.

Something like this should help clean this up. You might want to add some null checks as well.
```suggestion
import { find, getOr } from 'lodash/fp';
...
      const props = response.researcherProperties;
      const authProp = find({'propertyKey':'eraAuthorized'})(props);
      const expProp = find({'propertyKey':'eraExpiration'})(props);
      const isAuthorized = getOr(false,'propertyValue')(authProp);
      const expirationCount = AuthenticateNIH.expirationCount(getOr(0,'propertyValue')(expProp));
```